src/lightning_app/components/database/utilities.py

```diff
@@ -224,7 +224,7 @@ def __call__(self, data: Dict, response: Response):
             return {"status": "failure", "reason": "Unauthorized request to the database."}
 
         with Session(engine) as session:
-            update_data: SQLModel = self.models[data["cls_name"]].parse_raw(data["data"])
+            update_data: Type["SQLModel"] = self.models[data["cls_name"]].parse_raw(data["data"])
```

Comment:
I don't think these changes are correct, because if it was really a Type, then `update_data.__class__` would break below

Reply:
good point, moving these changes to #15242